x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/middle_truncated_text.tsx

```diff
@@ -28,27 +38,38 @@ const OuterContainer = styled.span`
   }
 `; // NOTE: min-width: 0 ensures flexbox and no-wrap children can co-exist
 
-const InnerContainer = styled.span`
+const InnerContainer = euiStyled.span`
   overflow: hidden;
   display: flex;
   align-items: center;
 `;
 
-const FirstChunk = styled.span`
+const IndexNumber = euiStyled(EuiText)`
+  margin-right: 3px;
+  line-height: ${FIXED_AXIS_HEIGHT}px;
+  text-align: right;
+  background-color: ${(props) => props.theme.eui.euiColorLightestShade};
```

Comment:
I'd like to suggest that we switch to use the EuiCodeFont to display the line numbers in a monospaced way to avoid the numbers changing size as you scroll down the page. This was also described in the original design solution. It also gives that line number style that you see in many other code displays.

Also changed the fixed value of `margin-right` attribute to use the provided size variables from EUI. It changes to `4px` but visually I'm not seeing much difference compared to `3px`.

<img width="379" alt="Screenshot 2021-05-27 at 13 24 53" src="https://user-images.githubusercontent.com/4104278/119818206-34f71000-beef-11eb-86a9-e77b414f5683.png">

```suggestion
const IndexNumber = euiStyled(EuiText)`
  font-family: ${(props) => props.theme.eui.euiCodeFontFamily};
  margin-right: ${(props) => props.theme.eui.euiSizeXS};
  line-height: ${FIXED_AXIS_HEIGHT}px;
  text-align: right;
  background-color: ${(props) => props.theme.eui.euiColorLightestShade};
```